packages/patrol_cli/lib/src/dart_defines_reader.dart

```diff
@@ -25,6 +27,19 @@ class DartDefinesReader {
     return _parse(lines);
   }
 
+  Map<String, dynamic> fromConfigFile({required String path}) {
+    final filePath = _fs.path.join(_projectRoot.path, path);
+    final file = _fs.file(filePath);
+
+    if (!file.existsSync()) {
```

Comment:
I think this is a very controversial decision. I can imagine this situation: you want to inject dart defines from a file and you make a typo in the path, so the file doesn't exist and your tests are being run but it just behaves in a different way than you expected.

This kind of behaviors can lead to hours-long debugging that gives you a headache but you just made a typo in a path.

BUT if we just gave a warning to the user, that this file doesn't exist, I would be so grateful as a user :D

Reply:
Ok, I've add logger warning